css/style.css

```diff
@@ -264,7 +264,62 @@ hr {
     margin-top: 50px;
 }
 
+/******************Section-Sharpen your skills*****************/
+.section table{
```

Comment:
In similar way change other tags also not only table.

- th
- tr
- td
- a
- hr
- h1